app/presenters/queries/available_translations.rb

```diff
@@ -32,7 +32,7 @@ def web_item(id)
       end
 
       def expanded_translations
-        @expanded_translations ||= grouped_translations.map do |_, (id)|
+        @expanded_translations ||= grouped_translations.map do |_, ((id, _, _))|
```

Comment:
We do need the extra brackets here.

What was happening before was that an array like `[1, "en", "draft"]` was being passed to `expand_translation` and then Rails was running a query like `id IN (1, 'en', 'draft')`. This happens because `grouped_translations` returns a hash mapping locale to an array containing one item which is the array containing id, locale and state. An alternative fix might be to change `grouped_translations` to not return the superfluous one item array?